aws/cloudformation-templates/base/_template.yaml

```diff
@@ -102,6 +102,52 @@ Parameters:
     Type: String
     Description: The GitHub branch used for deployment.
 
+  FenixZipDetectUrl:
+    Type: String
+    Description: Fenix Commerce Zipcode Detect URL
+    Default: 'https://ipapi.co/json?key=cKGC3jQbSIoXYmI2KtXObugsKfosD9Yr0HnkHhPUu1SM2wQhE0'
```

Comment:
is there not any risk of having publicly available keys like this?
Do we have the agreement from these 3rd parties
- ipapi
- fenixcommerce

if not, there should be instructions for the user to request api keys and add them there

Reply:
Fenix has agreement with ipapi and key used here is has rate limit with quota to prevent over use. this is for demo only key. Future version will have Fenix account creation which will give specific key for account.